src/test/java/uk/gov/hmcts/reform/wataskconfigurationtemplate/dmn/CamundaTaskWaInitiationTest.java

```diff
@@ -121,16 +118,8 @@ void should_return_outcome_for_delay_until_interval_in_dmn() {
         ), equalTo(delayUntil));
     }
 
-    private static LinkedHashMap<String, Object> sortMap(Map<String, Object> delayUntilIntervalDays) {
-        return delayUntilIntervalDays.entrySet().stream()
-            .sorted(Comparator.comparing(Map.Entry::getKey))
-            .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue, (e1, e2) -> e1, LinkedHashMap::new));
-    }
-
```

Comment:
guess this was unused code?

Reply:
Yes. Trawled through and not used anywhere. Reran tests after removal and all passed with no adverse effect.

Reply:
Nice one thanks